CHANGELOG.md

```diff
@@ -10,5 +10,9 @@ Adapters are compatible with any Chartboost Mediation SDK version within that ma
 ### 4.4.6.0.0
 - This version of the adapters has been certified with AmazonPublisherServicesSDK 4.6.0.
 
+### 4.4.5.6.1
+- Add support for Rewarded Video ad format.
```

Comment:
Both `4.4.6.0.1` and `4.4.5.6.1` have the entry for Rewarded Video. However since that change didn't happen linearly, I think it makes sense to have it exist twice in the changelog. Thoughts?

Reply:
Yeah let it exist twice

Reply:
Yeah lets have it in the changelog twice. I think accurately listing the changes under patch releases is a better compromise than trying to make the changelog perfectly linear.